### ISSUE TITLE
a11y: landmarks and page structure

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#774]
**What Should Happen**
Landmarks - `<header>`, `<nav>`, `<main>`, `<aside>`, `<footer>` - should be used correctly.  `<section>` should be used to structure content.  Every page should have `<header>`, `<nav>`, `<main>`; only one header and main should be present, but multiple nav elements can be used (e.g. if parts of the interface have specific navigational controls).

**JIRA Issues**
- [x] MOVE-118
- [x] MOVE-86
- [x] MOVE-64
- [x] MOVE-39
- [x] MOVE-6
